common/eth2_network_config/built_in_network_configs/gnosis/config.yaml

```diff
@@ -0,0 +1,85 @@
+# Gnosis Beacon Chain config
+
+# Extends the gnosis preset
+PRESET_BASE: 'gnosis'
+
+# Transition
+# ---------------------------------------------------------------
+# TBD, 2**256-2**10 is a placeholder
+TERMINAL_TOTAL_DIFFICULTY: 115792089237316195423570985008687907853269984665640564039457584007913129638912
+# By default, don't use these params
+TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
+TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
+
+# Genesis
+# ---------------------------------------------------------------
+# `2**12` (= 4,096)
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 4096
+# Dec 8, 2021, 13:00 UTC
+MIN_GENESIS_TIME: 1638968400
+# Gnosis Beacon Chain initial fork version
+GENESIS_FORK_VERSION: 0x00000064
+# 6000 seconds (100 minutes)
+GENESIS_DELAY: 6000
+
+
+# Forking
+# ---------------------------------------------------------------
+# Some forks are disabled for now:
+#  - These may be re-assigned to another fork-version later
+#  - Temporarily set to max uint64 value: 2**64 - 1
+
+# Altair
+ALTAIR_FORK_VERSION: 0x01000064
+ALTAIR_FORK_EPOCH: 512
+# Merge
+MERGE_FORK_VERSION: 0x02000064
```

Comment:
```suggestion
BELLATRIX_FORK_VERSION: 0x02000064
```

Reply:
We snuck this rename in recently in ef7351ddfecbb35cd23dde11835d58d4d4e689a8, you'll need to rebase to bring it in